pmd-plsql/src/test/resources/net/sourceforge/pmd/lang/plsql/ast/CreateOrganizedTable.pls

```diff
@@ -0,0 +1,252 @@
+--
+-- BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+--
+
+-- FIXED records
+CREATE TABLE emp_load (first_name CHAR(15), last_name CHAR(20), year_of_birth CHAR(4))
+  ORGANIZATION EXTERNAL (TYPE ORACLE_LOADER DEFAULT DIRECTORY ext_tab_dir
```

Comment:
Be aware, that due to the `Skip2NextOccurence` everything beginning with `ORGANIZATION` won't be available in the AST. E.g. this create table statement is parsed as this:

```
Table
  + ObjectNameDeclaration("emp_load")
  + TableColumn
    + ID("first_name")
    + ...
  + TableColumn
    + ID("last_name")
    + ...
  + TableColumn
    + ID("year_of_birth")
    + ...
  + Skip2NextOccurrence
```